packages/expect/src/index.js

```diff
@@ -259,9 +263,13 @@ expect.extend = (matchers: MatchersObject): void =>
 expect.anything = anything;
 expect.any = any;
 expect.objectContaining = objectContaining;
+expect.objectNotContaining = objectNotContaining;
```

Comment:
So we don't want to have `expect.not.objectContaining`?

Reply:
Yeah, I prefer to do `expect.not.objectContaining` as well since that is now expect works normally. I think it would create API inconsistency if we didn't do this.

Reply:
We could just pass the "not" argument in constructor and reuse existing classes, instead of creating new ones extending the old.